- Group invite

AmonDeShir opened the pull request:
This pull request adds a new page that allows administrators to invite users from the ranking to participate in the next quiz. 

## PC
![image](https://github.com/user-attachments/assets/e46d9d24-7357-4ce6-bc89-065aff3ff162)
![image](https://github.com/user-attachments/assets/d7f3308e-8fd1-45e3-9c63-4f0c6b99bb0e)
![image](https://github.com/user-attachments/assets/d290966c-1af6-459c-b5b9-67046a65ac28)


## Mobile
![image](https://github.com/user-attachments/assets/300efe8f-16eb-49ea-a4e8-024e39f87a61)
![image](https://github.com/user-attachments/assets/538e0fb9-8e71-4a6d-bf95-bff909e4e22b)

EwelinaSkrzypacz reviewed (commented):
1. As a user, when I go to ranking, I don't see the title of quiz (also page title is missing)

![image](https://github.com/user-attachments/assets/7da6cf0f-8279-47aa-8b56-78d971108302)

![image](https://github.com/user-attachments/assets/d181d5da-22ba-4416-8fec-af17c4b1f0ab)

2. Typo - it should be "Wycofaj publikację"

![image](https://github.com/user-attachments/assets/0830aba3-840c-4c09-b9f3-4d4351b477ac)

3. I'm not convinced that "Brak" is the best word. We could enhance the user experience by incorporating an autocomplete feature or a dropdown menu that suggests options the user can click to select a test.

![image](https://github.com/user-attachments/assets/194cc180-2d28-4f33-b00f-1166ea4acc7b)

AmonDeShir commented:
1. fixed
![image](https://github.com/user-attachments/assets/7f084e2e-d61b-4437-a8a8-bbd4f55c6a1c)
2. fixed

AmonDeShir commented:
1. fixed
![image](https://github.com/user-attachments/assets/7f084e2e-d61b-4437-a8a8-bbd4f55c6a1c)
2. fixed
![image](https://github.com/user-attachments/assets/a7b2c7cf-9de3-429d-8324-a97a76e864f3)
